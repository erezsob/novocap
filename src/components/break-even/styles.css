.container {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    align-items: center;
    height: 20px;
    width: 90%;
    padding-top: 20px;
    color: white;
    font-weight: bold;
    font-size: 20px;
}

.barContainer {
    margin: 20px;
    width: 150px;
    height: 150px;
    position: relative;
    display: flex;
    flex-direction: column;
    justify-content: center;
}

.title {
    display: flex;
    flex-direction: column;
    padding-top: 13px;
}

.date {
    font-size: 11px;
    font-weight: 300;
}

.addCosts {
    color: #fff;
    font-size: 19px;
}