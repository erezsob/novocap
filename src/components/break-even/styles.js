import { blueMain } from '../../common/variables.css';

export default {
  card: {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    boxShadow: 'none',
    height: 208,
    backgroundColor: blueMain,
    cursor: 'pointer'
  },
  iconButton: {
    color: 'white',
    width: 24
  }
};
