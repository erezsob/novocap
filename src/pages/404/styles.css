@value blueDark, blueText from './../../common/variables.css';

.container {
    display: flex;
    height: 100vh;
    align-items: center;
    justify-content: center;
}

.textBox {
    text-align: center;
    color: blueText;
    width: 273px;
    min-height: 260px;
}

.bigText {
    font-size: 800%;
    color: blueDark;
}
