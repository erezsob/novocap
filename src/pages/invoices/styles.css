@value bpSMedium from './../../common/variables.css';

.controls {
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: space-between;
    margin-top: 20px;
}

.controls > ul {
    padding: 20px 0 10px;
}

.controlsTop {
    composes: controls;
    justify-content: flex-end;
    margin: 0 0 10px;
}

.controlsTop > ul {
    padding: 10px 0;
}

@media(min-width: bpSMedium) and (min-device-width: bpSMedium) {
    .controls {
        height: 40px;
        flex-direction: row;
        padding: 20px 0;
        margin: 0;
    }

    .controls > * {
        padding: 0;
    }

    .controlsTop > ul {
        margin-bottom: 0;
    }
}
