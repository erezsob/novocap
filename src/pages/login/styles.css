@value washedOutWhite, red200, bpXSmall, bpMedium from './../../common/variables.css';

.container {
    display: flex;
    height: 100vh;
    align-items: center;
    justify-content: center;
    background: transparent url(../../img/login_bg.jpg) 20% center no-repeat;
    background-attachment: fixed;
    background-size: cover;
    position: relative;
}

@media (min-width: bpMedium) and (min-device-width: bpMedium) {
    .container {
        background-position-x: center;
    }
}

.loginBox {
    display: flex;
    align-items: center;
    flex-direction: column;
    background-color: washedOutWhite;
    width: 100%;
    height: 100%;
    justify-content: center;
    opacity: 0;
    transform: translateY(100px);
    transition: opacity 0.7s cubic-bezier(0.39, 0.575, 0.565, 1), transform 0.5s cubic-bezier(0.39, 0.575, 0.565, 1);
}

@media (min-width: bpXSmall) and (min-device-width: bpXSmall) {
    .loginBox {
        width: 273px;
        height: auto;
        min-height: 260px;
        justify-content: space-around;
        border-radius: 20px;
        padding: 26px 36px;
    }
}

.logo {
    max-width: 200px;
}

.boxEntered {
    opacity: 1;
}

.boxEntering {
    opacity: 0.01;
}

.boxEntered {
    transform: translateY(0);
}