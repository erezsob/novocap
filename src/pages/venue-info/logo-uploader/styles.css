@value defaultText, blueLight, offWhite, newBlack from '../../../common/variables.css';

.cloudLogo {
    color: lightGray;
}

.text1 {
    font-size: 16px;
    line-height: 20px;
    font-weight: bold;
    color: newBlack;
}

.text2,
.text3 {
    font-size: 11px;
    line-height: 14px;
}

.text2 {
    color: defaultText;
}

.text3 {
    color: blueLight;
}

.dashedBox {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    border-radius: 4px;
    border: 2px dashed offWhite;
    text-align: center;
    height: auto;
    padding: 10px 0;
}

@media (min-width: 960px) and (min-device-width: 960px) {
    .dashedBox {
        height: 245px;
        padding: 0;
    }
}