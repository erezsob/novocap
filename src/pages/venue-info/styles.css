.roundedWhiteBox {
    background-color: white;
    border-radius: 4px;
    padding: 12px 24px;
    width: 100%;
    box-sizing: border-box;
}

.group {
    margin-top: 10px;
}

.wrapper {
    display: flex;
    flex-wrap: wrap;
    width: 100%;
    box-sizing: border-box;
    margin: 0 -12px;
}

.extraPadding {
    padding: 12px 10px 0;
    width: 100%;
}

.map {
    border: 0;
    width: 100%;
    height: 245px;
}

.container {
    padding: 12px;
    display: flex;
    flex-wrap: wrap;
    flex: 0 0 auto;
    box-sizing: border-box;
}

.container50 {
    composes: container;
    flex-basis: 100%;
    max-width: 100%;
}

.container100 {
    composes: container;
    flex-basis: 100%;
    max-width: 100%;
}

@media (min-width: 960px) and (min-device-width: 960px) {
    .container50 {
        flex-basis: 50%;
        max-width: 50%;
    }

    .extraPadding {
        padding: 12px 0 0;
    }
}
