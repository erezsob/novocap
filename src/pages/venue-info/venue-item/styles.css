@value defaultText, newBlack, gray74 from '../../../common/variables.css';

.ellipsis {
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
}

.box {
    composes: roundedWhiteBox from '../styles.css';
}

.label {
    composes: ellipsis;
    color: newBlack;
    font-size: 12px;
    font-weight: bold;
    margin-bottom: 10px;
}

.field {
    composes: ellipsis;
    font-size: 16px;
    height: 16px;
    color: defaultText;
    padding-bottom: 10px;
    border-bottom: 1px solid gray74;
}
