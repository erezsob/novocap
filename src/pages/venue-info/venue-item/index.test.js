import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import VenueItem from './';
import styles from './styles.css';

describe('<VenueItem />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<VenueItem />);
  });

  it('renders properly', () => {
    const tree = renderer.create(<VenueItem />);
    expect(tree.toJSON()).toMatchSnapshot();
    expect(wrapper).toHaveLength(1);
  });

  it('renders the `label` and `value` passed as props', () => {
    wrapper.setProps({ label: 'foo', value: 'bar' });
    expect(wrapper.find(`.${styles.label}`).text()).toBe('foo');
    expect(wrapper.find(`.${styles.field}`).text()).toBe('bar');
  });
});
