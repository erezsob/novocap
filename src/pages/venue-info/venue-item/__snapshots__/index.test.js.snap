// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VenueItem /> renders properly 1`] = `
<div
  className="box"
>
  <div
    className="label"
  />
  <div
    className="field"
  />
</div>
`;
